File: mobile/src/main/java/net/maiatoday/levelbest/helpers/PreferenceHelper.java
package net.maiatoday.levelbest.helpers;

import android.content.SharedPreferences;

/**
 * Created by maia on 2016/07/25.
 */

public class PreferenceHelper {
    public static final String SHARED_PREF_NAME = "LevelBest";
    public static final String KEY_FIRST_TIME = "first_time";

    public static void write(SharedPreferences prefs, String key, boolean value) {
        SharedPreferences.Editor editor = prefs.edit();
        editor.putBoolean(key, value);
        editor.apply();
    }
}

File: mobile/src/main/kotlin/net/maiatoday/levelbest/helpers/PreferenceHelper.kt
package net.maiatoday.levelbest.helpers

import android.content.SharedPreferences

/**
 * Created by maia on 2016/07/25.
 */

object PreferenceHelper {
    val SHARED_PREF_NAME = "LevelBest"
    val KEY_FIRST_TIME = "first_time"

    fun write(prefs: SharedPreferences, key: String, value: Boolean) {
        val editor = prefs.edit()
        editor.putBoolean(key, value)
        editor.apply()
    }
}
